extensions/map-layers/package.json

```diff
@@ -69,14 +69,14 @@
     "chai-spies": "1.0.0",
     "cpx": "^1.5.0",
     "enzyme": "^3.4.0",
-    "enzyme-adapter-react-16": "^1.15.5",
+    "@wojtekmaj/enzyme-adapter-react-17": "^0.6.3",
```

Comment:
Did someone have to fork the package to get an update? Or is a PR pending on the main repo for them to support react 17?

Reply:
That's just an adapter for enzyme testing, not for anything in our libraries. I don't think it affects our consumers, does it?
It's written by the enzyme developer responsible for enzyme/react, but he's not ready to actually release. I rewrote any problematic enzyme tests using react-testing-library, but did  not take the time to refactor the ones that were working.